ProcessStore/src/main/java/at/fhjoanneum/ippr/processstore/persistence/entities/ProcessStoreObjectImpl.java

```diff
@@ -0,0 +1,102 @@
+package at.fhjoanneum.ippr.processstore.persistence.entities;
+
+
+import at.fhjoanneum.ippr.processstore.persistence.objects.ProcessStoreObject;
+import org.hibernate.validator.constraints.NotBlank;
+
+import javax.persistence.*;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.Date;
+
+@Entity(name="PROCESSSTORE")
+@XmlRootElement
+public class ProcessStoreObjectImpl implements ProcessStoreObject {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long processId;
+
+    @Column
+    @NotBlank
+    private String processName;
+
+    @Column
+    @NotBlank
+    private String processDescription;
+
+    @Column
+    @NotBlank
+    private String processCreator;
```

Comment:
Should this not be a `OneToOne` relationship with a `User` entity from the `Gateway` service?
We should definitely make a relationship here as this is a must.